test/functionalTest/cases/719_servicepath_for_registrations/cm_forwards_cb_with_servicePath.test

```diff
@@ -0,0 +1,456 @@
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
After looking to all the .test, not sure if we cover this functionality (from issue #719):

"In the case of "locally failed" query, it is more complex, as the Fiware-ServicePath allowed in queryContext can be a comma-separated lists of elements that, in sequence, could include sub-tree search with "#"). Probably the most coherent solution is that the "search space" be the one covered by que queryContext servicePath (taking into acocunt sequences and sub-trees)."

I miss some test with queryContext using:

* A single meaningfull servicePath "#"-based (by meaningfll I mean e.g., registration in "/A/A1" and queryContext in "/A/#")
* A list of 2 regular servicePaths
* A list of 2 elements, one of them a meaningufll "#" and the other a regular servicePath